package.yaml

```diff
@@ -31,13 +31,15 @@ dependencies:
 - filepath == 1.4.*
 - gitrev == 1.3.*
 - hashable == 1.4.*
+- language-c == 0.9.*
 - megaparsec == 9.2.*
 - microlens-platform == 0.4.*
 - parser-combinators == 1.3.*
 - polysemy == 1.7.*
 - polysemy-plugin == 0.4.*
 - prettyprinter == 1.7.*
 - prettyprinter-ansi-terminal == 1.1.*
+- pretty == 1.1.3.*
```

Comment:
Why is this really needed?

Reply:
Yes - it is the pretty printer used by the language-c package that is used by the C code serialiser: https://github.com/heliaxdev/minijuvix/blob/ee6eac9e4992c83ddc80e55c65ab5605966db858/src/MiniJuvix/Syntax/CJuvix/Serialization.hs#L9